### PR TITLE
incorporated additional feedback

### PR DESCRIPTION
**Pull request by rhoads-zach**

### CODE REVIEW

File: docs/topics/concept-getting-started-intro.adoc [hunk 1]
```diff
@@ -2,16 +2,38 @@
...
+The processes and practices used to do cloud-native development are implemented in software, not just defined.
+
+Containers::
+Containers are used as deployment targets for your cloud-native applications.
+
+Service-based Architecture::
+Cloud-native applications use a modular and loosely-coupled architecture such as microservices.
+
+Automation::
+Cloud-native applications are deployed through pipelines using instrumentation.
```
Comment (Ladicek) [inline docs/topics/concept-getting-started-intro.adoc]: This seems to duplicate the "DevOps" item above. Also not sure what "using instrumentation" means.

**Comment by rhoads-zach** [comment]
thanks @Ladicek, tried to clarify the difference between the two

**Comment by rhoads-zach** [comment]
merging.

**Comment by rhoads-zach** [comment]
merged: 3263dd80b4ed5f5485f6bf3cc8f900de00cd10cf

**Pull request closed without merging**